app/views/reports/regions/details.html.erb

```diff
@@ -59,16 +59,16 @@
                 Total
               </td>
               <td class="row-total ta-center">
-                <%= dash_if_zero(@dashboard_analytics.sum { |_, row| row[:total_registered_patients] || 0 }) %>
```

Comment:
We're no longer using dashes if the number is zero? Is that intentional?

Reply:
I did this intentionally, as it's the only table where we do this. I will get Daniel's approval first though.

Reply:
Looks like we want to stick with dashes according to Slack.

I did a quick check and it's safe to pass dashes to the `number_with_delimiter` methods, so we can update these lines to:
```erb
<%= number_with_delimiter(dash_if_zero(@dashboard_analytics.blah_blah)) %>
```

Reply:
Great, thank you @harimohanraj89 !